ui/de-lib/src/main/java/org/iplantc/de/admin/desktop/client/ontologies/views/OntologiesViewImpl.java

```diff
@@ -276,18 +300,14 @@ public void onSelection(SelectionEvent<Ontology> event) {
         return ontologySimpleComboBox;
     }
 
-    void updateButtonStatus() {
+    public void updateButtonStatus() {
         publishButton.setEnabled(selectedOntology != null && selectedOntology != activeOntology);
         saveHierarchy.setEnabled(selectedOntology != null);
+        deleteButton.setEnabled(selectedOntology != null && selectedOntology != activeOntology);
+        deleteHierarchy.setEnabled(selectedOntology != null && treeStore.getRootItems().size() > 0);
```

Comment:
I think this should be OK. Especially since the admins might only ever add 1 ontology, so they should be able to modify the hierarchies for the sole, active ontology.